command/sub8_alarm/sub8_alarm/alarm_helpers.py

```diff
@@ -0,0 +1,80 @@
+import rospy
+from sub8_msgs.msg import Alarm
+from std_msgs.msg import Header
+import json
+
+class AlarmBroadcaster(object):
+    def __init__(self):
+        '''Alarm Broadcaster'''
+        # Notify the world when our alarm happens
+        self.alarm_pub = rospy.Publisher('/alarm', Alarm, latch=True, queue_size=10)
+
+        # Notify the world that our alarm has spawned
+        # For now, unused - future: Use for system reports
+        # self.alarm_spawn_pub = rospy.Publisher('/alarm_spawn', Alarm, latch=True, queue_size=1)
+
+        # Get ROS name
+        self.node_name = rospy.get_name()
+
+        # Alarms associated with this broadcaster
+        # In the future, there will be some background bookkeeping
+        self.alarms = []
+
+    def add_alarm(self, name, action_required=False, severity=5, problem_description=None, parameters=None):
```

Comment:
Ohp, changing the code. I did this because I glanced at the wiki, and the
biggest number I saw was "5"

Jacob Panikulam | (630) - 877 - 7665
Software Lead, 2016 Subjugator Team <http://subjugator.org/>
Github <https://github.com/jpanikulam> | About Me
<http://jpanikulam.github.io/>

On Mon, Nov 9, 2015 at 1:40 PM, Patrick E. <notifications@github.com> wrote:

> In command/sub8_alarm/sub8_alarm/alarm_helpers.py
> <https://github.com/uf-mil/Sub8/pull/23#discussion_r44312162>:
>
> > +        '''Alarm Broadcaster'''
> > +        # Notify the world when our alarm happens
> > +        self.alarm_pub = rospy.Publisher('/alarm', Alarm, latch=True, queue_size=10)
> > +
> > +        # Notify the world that our alarm has spawned
> > +        # For now, unused - future: Use for system reports
> > +        # self.alarm_spawn_pub = rospy.Publisher('/alarm_spawn', Alarm, latch=True, queue_size=1)
> > +
> > +        # Get ROS name
> > +        self.node_name = rospy.get_name()
> > +
> > +        # Alarms associated with this broadcaster
> > +        # In the future, there will be some background bookkeeping
> > +        self.alarms = []
> > +
> > +    def add_alarm(self, name, action_required=False, severity=5, problem_description=None, parameters=None):
>
> Update wiki with severity levels; the wiki currently only has descriptions
> for levels 0-2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uf-mil/Sub8/pull/23/files#r44312162>.
>